vent/common/prefs.py

```diff
@@ -64,6 +67,17 @@ def set_pref(key: str, val):
     if globals()['LOADED'].value == True:
         save_prefs()
 
+def set_time():
```

Comment:
I would rather this use the getter/setter methods and be part of the load_prefs method, 

so in the load_prefs method something like

```
if get_pref('TIME_FIRST_START') is None:
   set_pref('TIME_FIRST_START', time.time()) 
```

which always ensures it's set, doesn't require new method, and doesn't need an ectopic call elsewhere in the code.  I think this will probably work, but i'd like to keep all the common modules pretty simple and static from here on out

Reply:
done